docs/sources/k6/next/using-k6/metrics/_index.md

```diff
@@ -67,11 +67,12 @@ The preceding script outputs something like this:
 ```bash
 $ k6 run script.js
 
-          /\      |‾‾| /‾‾/   /‾‾/
-     /\  /  \     |  |/  /   /  /
-    /  \/    \    |     (   /   ‾‾\
-   /          \   |  |\  \ |  (‾)  |
-  / __________ \  |__| \__\ \_____/ .io
+         /\      Grafana   /‾‾/
+    /\  /  \     |\  __   /  /
+   /  \/    \    | |/ /  /   ‾‾\
+  /          \   |   (  |  (‾)  |
+ / __________ \  |_|\_\  \_____/
```

Comment:
Thanks for updating this! I did a quick search and found some other instances, I'm going to make a commit to update them as well.